<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWindow</class>
 <widget class="QDialog" name="SignupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="usernameInput">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
   <widget class="QLineEdit" name="passwordInput">
     <property name="text">
      <string/>
     </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   </item>
   <item>
   <widget class="QLineEdit" name="confirmPasswordInput">
     <property name="text">
      <string/>
     </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   </item>
   <item>
    <widget class="QPushButton" name="signupBtn">
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
